src/components/AzureMapFeature/AzureMapFeature.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState, useContext } from 'react'
+import { IAzureMapFeature, IAzureMapDataSourceProps } from '../../types'
+import atlas from 'azure-maps-control'
+import { AzureMapDataSourceContext } from '../../contexts/AzureMapDataSourceContext'
+
+const createFeature = ({
+  type,
+  coordinate,
+  coordinates,
+  multipleCoordinates,
+  multipleDimensionCoordinates,
+  bbox
+}: IAzureMapFeature): atlas.data.Geometry | undefined => {
```

Comment:
undefined?

Reply:
It is coming from helper function, we can make it as null